File: src/main/java/cn/zcc/vastsea/VastseaLuck.java
package cn.zcc.vastsea;

import net.md_5.bungee.api.ProxyServer;
import net.md_5.bungee.api.plugin.Plugin;

public class VastseaLuck extends Plugin {
    public static VastseaLuck vastseaLuck;

    @Override
    public void onLoad() {
        vastseaLuck = this;
        getLogger().info("Luck!!!~");
    }

    @Override
    public void onEnable() {
        ProxyServer.getInstance().getPluginManager().registerCommand(this, new CommandRuntime("luck"));
    }
}

File: src/main/kotlin/cn/zcc/VastseaLuck.kt
package cn.zcc

import net.md_5.bungee.api.plugin.Plugin

class VastseaLuck : Plugin() {
    companion object {
        @JvmStatic
        lateinit var vastseaLuck: VastseaLuck
    }

    override fun onLoad() {
        vastseaLuck = this
    }

    override fun onEnable() {

    }

}